contracts/stake/src/contract/transaction.rs

```diff
@@ -112,4 +112,17 @@ impl StakeContract {
 
         (id, transaction)
     }
+
+    pub fn whitelist_transaction(
+        pk: PublicKey,
+        signature: Signature,
+        owner: PublicKey,
+    ) -> (ContractId, Transaction) {
+        let id = rusk_abi::stake_contract();
+
+        let transaction = (TX_WHITELIST, pk, signature, owner);
+        let transaction = Transaction::from_canon(&transaction);
```

Comment:
Would it be better to place the signature at the end of the buffer? I might make it easier to parse both the owner and the key being whitelisted.